README.md

```diff
@@ -23,6 +23,8 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 > For in-depth development and usage notes, see the [Contribution Guidelines](https://github.com/quantumblacklabs/kedro-viz/blob/main/CONTRIBUTING.md).
 
+> It's likely that Kedro-Viz>=3.8.0 will not work with projects created with older versions of Kedro<=0.16.6. Please migrate your project to Kedro>=0.17.0 before installation of the latest version of Kedro-Viz.
```

Comment:
```suggestion
> It's likely that Kedro-Viz>=3.8.0 will not work with projects created with older versions of Kedro<=0.16.6. Please migrate your project to Kedro>=0.17.0 before you install the latest version of Kedro-Viz.
```

Reply:
"It's likely..." is one of those terms that fills me with concern. Will it? Won't it? How will I know? I think we need to be definite and say it won't.

Reply:
thank you for raising this @stichbury ! From our testing, Kedro-Viz 3.8.0 **will** work with older versions of Kedro, however the problem lies with many users running latest Kedro viz on Kedro 17 against 

Regardless, for the wording I had taken the exact same wording that was in the Kedro release notes ( please see below conversation on slack) - my approach here is to follow what was stated in the kedro-release notes ( given that this incompatibility was not introduced by Kedro-viz but rather with the new Kedro upgrades.) The point of including this note here is just merely to reinterate / plug the gap. 

![image](https://user-images.githubusercontent.com/27775658/107647056-9b269280-6c72-11eb-8628-8512ff369e83.png)

I am not sure we can say it won't since actually it **would** work on older versions of Kedro ( provided it's on Kedro 16 with Kedro 16 project), but it could likely won't because of the comparibility between the kedro project and kedro. @yetudada @stichbury would love your thoughts on how I should proceed with this sticky situation?